backend/src/packages/trucks/truck.repository.ts

```diff
@@ -83,6 +88,61 @@ class TruckRepository implements IRepository {
 
     return await preparedQuery.execute({ query: `%${query}%` });
   }
+
+  public async getTrucksByBusinessId(
+    id: number,
+  ): Promise<TruckDatabaseModel[]> {
+    return await this.db
+      .driver()
+      .select()
+      .from(this.trucksSchema)
+      .where(eq(this.trucksSchema.businessId, id));
+  }
+
+  public async addUser(
+    userId: number,
+    truckId: number,
+  ): Promise<
+    {
+      userId: number;
+      truckId: number;
+    }[]
+  > {
```

Comment:
type DriverHaveAccessToTruck = {
      userId: number;
      truckId: number;
 }
```suggestion
  public async addTruckToDriver({ userId, truckId}: DriverHaveAccessToTruck): Promise<void
  > {
```